### PR TITLE
Cinder Integration page : Fix code formatting and missing screenshots

### PR DESCRIPTION
**Pull request by rollandf**
Changes proposed in this pull request:

- Fix code formating

- Add missing screenshots

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @rollandf

This pull request needs review by: @mureinik @danielerez @maorlipchuk

### CODE REVIEW

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 1]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
-` `<name></name>
-` `<url><fqdn>`:8776`</url>
-` `<data_center>
-`   `<id|name></id|name>
-` `</data_center>
-` `<requires_authentication>`true|false`</requires_authentication>
-` `<username></username>
-` `<password></password>
-` `<tenant_name></tenant_name>
+    <name></name>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Why do we need an empty name?

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 2]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <url><fqdn>`:8776`</url>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 3]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <data_center>
+        <id|name></id|name>
+    </data_center>
+    <requires_authentication>`true|false`</requires_authentication>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 4]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
 
 #### Add Provider: POST /api/openstackvolumeproviders
 
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Probably worth using "```xml" to get better highlighting

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 5]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
-` `<openstack_volume_providers>
-`  `<openstack_volume_provider href="/api/openstackvolumeproviders/{id}" id="{id}">
-`    `<name>`cinder2`</name>
-`    `<requires_authentication>`true`</requires_authentication>
-`    `<username>`cinder`</username>
-`    `<data_center href="/api/datacenters/{id}" id="{id}">
+```
+<openstack_volume_providers>
+    <openstack_volume_provider href="/api/openstackvolumeproviders/{id}" id="{id}">
+        <name>`cinder2`</name>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 6]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+        <requires_authentication>`true`</requires_authentication>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 7]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+        <username>`cinder`</username>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 8]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 
 #### Get Volume Provider: GET /api/openstackvolumeproviders/{provider_id} (All-Content: true)
 
-` `<openstack_volume_providers>
-`  `<openstack_volume_provider href="/api/openstackvolumeproviders/{id}" id="{id}">
-`    `<name>`cinder2`</name>
-`    `<requires_authentication>`true`</requires_authentication>
-`    `<username>`cinder`</username>
-`    `<data_center href="/api/datacenters/{id}" id="{id}">
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Should probably use "```xml"

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 9]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
-`    `</data_center>
-`  `</openstack_volume_provider>
+        </data_center>
+    </openstack_volume_provider>
 </openstack_volume_providers>
+```
 
 #### Get Volume Type: GET /api/openstackvolumeproviders/{provider_id}/volumetypes
 
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Should probably use "```xml"

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 10]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
-`   `<properties>
-`     `<property>
-`       `<name>`volume_backend_name`</name>
-`       `<value>`ceph`</value>
-`    `</property>
-`  `</properties>
-`  `<openstack_volume_provider href="/api/openstackvolumeproviders/{provider_id}" id="{id}"/>
-` `</openstack_volume_type>
+    <openstack_volume_type href="/api/openstackvolumeproviders/{id}/volumetypes/{volume_type_id}" id="{id}">
+        <name>`ceph`</name>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 11]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+        <properties>
+            <property>
+            <name>`volume_backend_name`</name>
+            <value>`ceph`</value>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 12]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+            </property>
+        </properties>
+        <openstack_volume_provider href="/api/openstackvolumeproviders/{provider_id}" id="{id}"/>
+    </openstack_volume_type>
 </openstack_volume_types>
+```
 
 #### Get Authentication Keys: GET /api/openstackvolumeproviders/{provider_id}/authenticationkeys
 
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Use ```xml

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 13]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 <openstack_volume_authentication_keys>
-`  `<openstack_volume_authentication_key>
-`   `<description>`my ceph secret`</description>
-`   `<uuid>`c50352a3-0700-48e9-9189-ed359c09bcf8`</uuid>
-`   `<usage_type>`ceph`</usage_type>
-`   `<creation_date>`2015-05-31T15:28:25.525+03:00`</creation_date>
-` `</openstack_volume_authentication_key>
+    <openstack_volume_authentication_key>
+    <description>`my ceph secret`</description>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 14]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <uuid>`c50352a3-0700-48e9-9189-ed359c09bcf8`</uuid>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 15]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <usage_type>`ceph`</usage_type>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 16]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <creation_date>`2015-05-31T15:28:25.525+03:00`</creation_date>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 17]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+   </openstack_volume_authentication_key>
 </openstack_volume_authentication_keys>
+```
 
 #### Create an Authentication Key: POST /api/openstackvolumeproviders/{provider_id}/authenticationkeys
 
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Use ```xml

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 18]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 <openstack_volume_authentication_key>
-`  `<uuid>`0e6fff8d-8af9-49e2-b04f-1a5dbbe883a2`</uuid>
-`  `<description>`my ceph secret`</description>
-`  `<usage_type>`ceph`</usage_type>
-`  `<value>`YQo=`</value>
+    <uuid>`0e6fff8d-8af9-49e2-b04f-1a5dbbe883a2`</uuid>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 19]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <description>`my ceph secret`</description>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 20]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <usage_type>`ceph`</usage_type>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 21]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <value>`YQo=`</value>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 22]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
-`   `</openstack_volume_type>
-`   `<storage_domains>
-`     `<storage_domain>
-`       `<name>`cinder`</name>
-`     `</storage_domain>
-         `</storage_domains>`    
-`   `<provisioned_size>`1073741824`</provisioned_size>
-`   `<interface>`virtio`</interface>
-`   `<format>`raw`</format>
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Use ```xml

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 23]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+<disk>
+    <openstack_volume_type>
+        <name>`my_ceph`</name>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 24]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    </openstack_volume_type>
+    <storage_domains>
+        <storage_domain>
+            <name>`cinder`</name>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 25]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+        </storage_domain>
+    </storage_domains>`    
+    <provisioned_size>`1073741824`</provisioned_size>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 26]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <interface>`virtio`</interface>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 27]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <format>`raw`</format>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The `s here should probably also be removed

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 28]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 </disk>
+```
 
 #### Get Unregistered Disks: GET /api/storagedomains/{storage_domain_id}/disks;unregistered
 
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Use ```xml

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 29]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 <disks>
       ...
 </disks>
+```
 
 #### Register Disk: POST /api/storagedomains/{storage_domain_id}/disks;unregistered
 
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Use ```xml

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 30]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 #### Delete Entity (DIsk/VM/Template)
 
 Cinder disks are deleted asynchronously, hence ';async' flag could be passed as part of the URL for getting 202-Accepted return status.
 
-      E.g. 
-      DELETE /api/disks/{disk_id};async
+E.g. 
+    DELETE /api/disks/{disk_id};async
+    
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Use ```xml

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 31]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 <action></action>
+```
 
 ### VDSM
 
 *   Add [librbd1 package](https://apps.fedoraproject.org/packages/librbd1) as dependency to vdsm.spec file.
 *   Refactor 'Dirve -> getXML()' to support multiple hosts (represents Ceph monitors) in disk's source element:
 
+```
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Use ```xml

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 4 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 2 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 3 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 8 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 5 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 6 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 7 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 9 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 10 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 12 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 11 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 13 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 14 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 15 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 16 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 17 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 18 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 19 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 20 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 21 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 22 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 23 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 24 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 25 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 26 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 27 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 28 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 29 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 30 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md (hunk 31 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Done

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 32]
```diff
@@ -8,7 +8,7 @@ wiki_revision_count: 71
 wiki_last_updated: 2015-10-27
 feature_name: Cinder Integration
 feature_modules: engine/vdsm
-feature_status: Stabilization
+feature_status:  Released in oVirt 3.6
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: There's a redundant space before "Released"

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 33]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
 
 *   Add [librbd1 package](https://apps.fedoraproject.org/packages/librbd1) as dependency to vdsm.spec file.
 *   Refactor 'Dirve -> getXML()' to support multiple hosts (represents Ceph monitors) in disk's source element:
 
-<disk type='network' device='disk'>
-             
-
-`               `<host name='{monitor-host}' port='6789'/>
+```xml
+<disk type=network device=disk>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The values need to be surrounded with "s to be valid XML

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 34]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
+    <host name={monitor-host} port=6789/>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: The values need to be surrounded with "s to be valid XML

**Comment by mureinik** [comment]
There are still some XML issues, sorry for not noticing in the previous round - see above.

File: source/develop/release-management/features/storage/cinder-integration.html.md [hunk 35]
```diff
@@ -118,115 +118,133 @@ There is a known issue with OpenStack when deleting a snapshot which has depende
...
-`   `<id|name></id|name>
-` `</data_center>
-` `<requires_authentication>`true|false`</requires_authentication>
-` `<username></username>
-` `<password></password>
-` `<tenant_name></tenant_name>
+    <name></name>
+    <url><fqdn>:8776</url>
+    <data_center>
+        <name></name>
```
Comment (mureinik) [inline source/develop/release-management/features/storage/cinder-integration.html.md]: Question: why did you remove the `id`  from `data_center`?
